kuma/wiki/templates/wiki/revision.html

```diff
@@ -8,7 +8,7 @@
 {% block content %}
 
   <article>
-    <h1>{{ document.title }}</h1>
+    <h1><a href="{{ url('wiki.document', document.full_path, locale=document.locale) }}">{{ _('{title}')|fe(title=document.title) }}</a></h1>
```

Comment:
Sorry for the late reply.

> Bottom line is that document.title comes from users, so we don't trust it. We store any markup un-escaped, so it needs to be escaped in the template.
> 
> I.e., @SebastianZ - please revert the element content back to {{ document.title }}

Isn't the conclusion from your sentences above to rather keep the escaping? HTML needs to be escaped like for [<img>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img$revision/684477) for example. Note that https://github.com/mozilla/kuma/blob/dcb90e5263d432ea4a654910faf6d071f7f7dd82/kuma/wiki/templates/wiki/document_revisions.html#L14 is using the same code.

Sebastian